src/constants/DateFormat.ts

```diff
@@ -0,0 +1,4 @@
+export const DateFormat = {
```

Comment:
I think this should be dateFormat by our conventions.

Reply:
We seem to be using pascal case in Baz, I followed that. 
<img width="1718" alt="Screenshot 2024-12-23 at 16 47 27" src="https://github.com/user-attachments/assets/d3ce5a5c-500d-4e33-8ec4-aa423f89ab2a" />